operators/hpe-csi-operator/Makefile

```diff
@@ -68,7 +68,7 @@ prep: init
 	cp -f sources/LICENSE $(IMAGE_NAME)/LICENSE
 
 	# Community and OpenShift Operators
-	cd $(IMAGE_NAME); docker-buildx build --progress=plain --no-cache \
+	cd $(IMAGE_NAME); docker buildx build --progress=plain --no-cache \
```

Comment:
This will break most installs. Make a docker-buildx symlink to docker to keep widest compatibility.